src/betterproto/__init__.py

```diff
@@ -495,6 +588,34 @@ class ParsedField:
     raw: bytes
 
 
+def load_fields(stream: BinaryIO) -> Generator[ParsedField, None, None]:
```

Comment:
```suggestion
def load_fields(stream: BinaryIO) -> Iterator[ParsedField]:
```
- see https://docs.python.org/3/library/typing.html#typing.Generator

Reply:
Ah, right, thanks for pointing it out, I'd just used what was already there for parse_fields. Changed it now.